R/shell.R

```diff
@@ -18,6 +20,8 @@
 #' @export
 start_shell <- function(master,
                         spark_home = Sys.getenv("SPARK_HOME"),
+                        spark_version = NULL,
+                        hadoop_version = NULL,
```

Comment:
I think we can remove `hadoop_version` since it's not actually used by `start_shell` (we only require the spark_version so we can pass it to extensions for dependencies).